projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_interface.g4

```diff
@@ -204,6 +204,11 @@ if_ip_ospf_hello_interval
    IP OSPF HELLO_INTERVAL seconds = DEC NEWLINE
 ;
 
+if_ip_ospf_network
+:
+   IP OSPF NETWORK POINT_TO_POINT NEWLINE
```

Comment:
is this the only valid line? Otherwise, looks like a regression in the set of things we can handle / ignore.

Or is that also unneeded because the unrecognized line code will handle this anyway (in which case, should we just get rid of the `_null` rules)?

Reply:
`_null` rules are needed so lines don't get thrown out. Not every `_null` rule is an entire line. However those that are could potentially be removed or omitted in future.

Only handling `point-to-point` here is fine since we didn't do anything with other lines anyway.